packages/shared/src/components/fields/PasswordField.tsx

```diff
@@ -61,7 +62,7 @@ export function PasswordField({
       autoComplete="off"
       type={useType}
       valueChanged={onChange}
-      leftIcon={<LockIcon />}
+      leftIcon={<LockIcon size={IconSize.Small} />}
```

Comment:
Should this be not needed as the default is `Small` or is it getting overridden by the component?

Reply:
You're right, I migrated them all as they where 1 bigger so missed these now being the default value, let me adjust.

Reply:
Actually default is `IconSize.XSmall` now so this is still right.

Reply:
Oh, right. So the default previously was small. Then kindly disregard my comments pointing to default values.